tstypes/utilities/index.d.ts

```diff
@@ -104,10 +104,8 @@ export { assertValidName, isValidNameError } from './assertValidName';
 export {
   BreakingChangeType,
   DangerousChangeType,
-  findBreakingChanges,
```

Comment:
A lot of tooling currently relies on only getting the schema changes. I think all  the code changes from above are great, but until we make `findSchemaChanges` more flexible, we should keep these utility functions, to avoid forcing people to rewrite their code significantly in order to upgrade.